PaintTest/main.cpp

```diff
@@ -0,0 +1,209 @@
+#pragma region Copyright (c) 2014-2016 OpenRCT2 Developers
+/*****************************************************************************
+ * OpenRCT2, an open source clone of Roller Coaster Tycoon 2.
+ *
+ * OpenRCT2 is the work of many authors, a full list can be found in contributors.md
+ * For more information, visit https://github.com/OpenRCT2/OpenRCT2
+ *
+ * OpenRCT2 is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * A full copy of the GNU General Public License can be found in licence.txt
+ *****************************************************************************/
+#pragma endregion
+
+#include <algorithm>
+#include <string>
+#include <vector>
+
+extern "C" {
+#include "main.h"
+#include "../src/rct2.h"
+#include "../src/ride/ride.h"
+#include "../src/ride/ride_data.h"
+#include "../src/ride/track.h"
+#include "../src/ride/track_data.h"
+}
+
+typedef struct {
+	uint8 rideType;
+	std::vector<uint8> trackTypes;
+} TestCase;
+
+extern const utf8string RideNames[91];
```

Comment:
These are from a `.c` file they should be wrapped in an `extern "C"` otherwise the linker will look for the mangled names.